pages/learn/deploy/release-strategy/release-policy.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Release policy
+excerpt: Manage the application and device release policy
+---
+
+# Release policy
+
+When managing an application fleet, you may require devices to be running different releases. By default, new releases are deployed to all devices in the fleet once successfully built. However, you can customize this behavior, so the application fleet or individual devices remain on a fixed release by pinning them.
```

Comment:
maybe some how working in the term "rolling release" as that is what we usually refer to it as for the default policy

Reply:
Addressed via https://github.com/balena-io/docs/pull/1242/commits/7daa70d6f8f3586690f06434d7be41b59643e2ef